### DISCUSSION TITLE
Change license to LGPL

### DISCUSSION THREAD

**Discussion by JMG1** [JMG1/ExplodedAssembly#32] [Announcements]
Hi everybody, 
this is a discussion to change the license of this WB from GPL2.0 to LGPL2+, as is happening with other workbenches (see [this discussion](https://github.com/shaise/FreeCAD_SheetMetal/issues/286) ).

Any thoughts on this?

@luzpaz 
@morganrallen 
@EvanLF6768

**Comment by luzpaz** [comment]
no issues from me
